packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleBannerTests.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Linq.Expressions;
+using Bunit;
+using Xunit;
+
+namespace NimbleBlazor.Tests.Unit.Components;
+
+/// <summary>
+/// Tests for <see cref="NimbleBanner"/>.
+/// </summary>
+public class NimbleBannerTests
+{
+    [Fact]
+    public void NimbleBanner_Render_HasBannerMarkup()
+    {
+        var context = new TestContext();
+        context.JSInterop.Mode = JSRuntimeMode.Loose;
+        var expectedMarkup = "nimble-banner";
+
+        var banner = context.RenderComponent<NimbleBanner>();
+
+        Assert.Contains(expectedMarkup, banner.Markup);
+    }
+
+    [Theory]
+    [InlineData(BannerSeverity.Default, "<nimble-banner blazor:onnimblebannertoggle=\"1\">")]
```

Comment:
I think you should leave off the `blazor:onnimblebannertoggle=\"1\"` stuff. That's implementation details of how blazor implements event handling, we don't own that

Reply:
Fair enough. I changed the assertion to do a regex match instead of a straight string comparison.